README.md

```diff
@@ -25,29 +25,27 @@ https://fury.apache.org
 
 In addition to cross-language serialization, Fury also features at:
 
-- Drop-in replace Java serialization frameworks such as JDK/Kryo/Hessian, but 100x faster at most, which can greatly improve 
+- Drop-in replace Java serialization frameworks such as JDK/Kryo/Hessian, but 100x faster at most, which can greatly improve
  the efficiency of high-performance RPC calls, data transfer, and object persistence.
-- **100% compatible** with JDK serialization API with much faster implementation: supporting JDK `writeObject`/`readObject`/`writeReplace`/`readResolve`/`readObjectNoData`/`Externalizable` API. 
+- **100% compatible** with JDK serialization API with much faster implementation: supporting JDK `writeObject`/`readObject`/`writeReplace`/`readResolve`/`readObjectNoData`/`Externalizable` API.
 - Supports **Java 8~21**, Java 17+ `record` is supported too.
 - Supports [AOT compilation serialization](docs/guide/graalvm_guide.md) for **GraalVM native image**, and no reflection/serialization json config are needed.
 - Supports shared and circular reference object serialization for golang.
 - Supports [scala serialization](docs/guide/scala_guide.md)
 - Supports automatic object serialization for golang.
 
 ## Protocols
```

Comment:
Could we replace all `protocols` with `format`?

Reply:
@LiangliangSui I replaced `protocol` in body to `protocol`, but keep this header as `Protocols`.
Some users may search the readme by `protocol`/`format`, those two keywords should both hit the result.

And it's used simultaneously: 
- https://docs.oracle.com/javase/8/docs/platform/serialization/spec/protocol.html
- flatbuffer: https://flatbuffers.dev/md__internals.html
- hessian: https://www.caucho.com/resin-3.1/doc/hessian-2.0-spec.xtp

Reply:
Great! I got it.